client/checkout/blocks/upe-deferred-intent-creation/payment-processor.js

```diff
@@ -174,14 +169,23 @@ const PaymentProcessor = ( {
 					) {
 						return {
 							type: 'error',
-							message:
+							message: __(
 								'This payment method cannot be saved for future use.',
+								'woocommerce-payments'
+							),
 						};
 					}
 
-					await validateElements( elements );
+					try {
+						await validateElements( elements );
+					} catch ( e ) {
+						return {
+							type: 'error',
+							message: e.message,
+						};
+					}
 
-					const paymentMethodObject = await api
+					const result = await api
 						.getStripeForUPE( paymentMethodId )
 						.createPaymentMethod( {
```

Comment:
The `createPaymentMethod` on the Stripe API doesn't throw an error - it handles everything through the `.then`. So there is no need for a `try`/`catch` block, in this case.